task-sdk/dev/generate_task_sdk_models.py

```diff
@@ -75,6 +76,7 @@ def load_config():
     return cfg
 
 
+@mock.patch("fastapi._compat.v2.get_long_model_name", lambda model: model.__name__)
```

Comment:
FastAPI 0.119.0 introduces a new way to handle conflicting names in the openapi schema, this create super long ExecutionAPIAssetResponse etc...

This restores the old behavior while we figure out a better way of handling this.